index.js

```diff
@@ -0,0 +1,187 @@
+// Create the getFullNameFromUser function
+
+const user = {
+    firstName: 'John',
+    lastName: 'Smith'
+}
+function getFullNameFromUser(user) {
+    return user.firstName + ' ' + user.lastName;
+}
+
+console.log(getFullNameFromUser(user));
+
+// Write the createUser function
+
+user.firstName = 'Katie',
+user.lastName = 'Williams'
+
+function createUser(firstName, lastName) {
+    return {
+        fullName: firstName + ' ' + lastName,
+        firstName,
+        lastName
+    }
+}
+
+const userName = createUser(user.firstName, user.lastName);
+console.log(userName);
+
+// write the checkIfPropertyIsDefined function
+const userObject = {
+    height: 170
+}
+function checkIfPropertyIsDefined(userObject, propertyName) {
```

Comment:
The name of the `userObject` argument is clashing with the `userObject` variable.